index.html

```diff
@@ -0,0 +1,190 @@
+                  <a href="#">Logout</a>
+                </li>
+              </ul>
+            </div>
+            <div class="menu-main">
+              <div class="menu-img">
+                <img
+                  class="menu-button"
+                  src="resources/images/icon-menu.svg"
+                  alt="menu icon"
+                />
+                <img
+                  class="menu-logo"
+                  src="resources/images/logo.png"
+                  alt="logo"
+                />
+                <div>&nbsp;</div>
+              </div>
+              <ul id="menu-list" class="menu-list hidden">
+                <li class="menu-item">
+                  <a class="fg-primary" href="about.html">About</a>
+                </li>
+                <li class="menu-item"><a href="#">Program</a></li>
+                <li class="menu-item"><a href="#">Join</a></li>
+                <li class="menu-item"><a href="#">Sponsor</a></li>
+                <li class="menu-item"><a href="#">News</a></li>
+                <li class="menu-item">
+                  <a class="border-primary" href="mailto:sonaban.art@gmail.com"
+                    >Contact</a
+                  >
+                </li>
+              </ul>
+            </div>
+          </div>
+        </nav>
+      </header>
+      <section id="intro" class="section intro">
+        <h2 class="event-welcome text-gradient">¡Bienvenidos, tuiteros!</h2>
+        <h1 class="event-title text-gradient">EXPO tuiteratura<br />2022</h1>
+        <p class="event-message">
+          <em>Expo tuiteratura 2022</em> es una iniciativa de Ana Baños,
+          <a href="https://twitter.com/sonabana_art">@sonabana_art</a>, arropada
+          con entusiasmo por 200 noveles hilanderos, todos jóvenes
+          universitarios, estudiantes de 4to, 7mo y 9no semestre de la facultad
+          de Psicología - UANL.
+        </p>
+        <h4 class="event-dates">25 de abril al<br />15 de mayo de 2022</h4>
+        <h4 class="event-organizer">
+          <a href="mailto:sonaban.art@gmail.com">sonaban.art@gmail.com</a>,
+          <a href="https://twitter.com/sonabana_art">@sonabana_art</a>
+        </h4>
+      </section>
+
+      <section id="program" class="section program">
+        <h2 class="section-title">Programa</h2>
+        <div class="separator"></div>
+        <div class="program-container">
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_01.png" alt="icon-1" />
+            </div>
+            <div class="program-title">Conferencia</div>
+            <div class="program-description">
+              Excucha a los expositores de varios países compartiendo sus
+              experiencias.
+            </div>
+          </div>
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_02.png" alt="icon-1" />
+            </div>
```

Comment:
- [ ]  [Optional] kindly align your text in mobile view as it is overlapping 👇 
        
![image](https://user-images.githubusercontent.com/66097371/163557658-48a1e74c-cfa3-4cb6-8fff-75c50602b265.png)